spec/controllers/blog_controller_spec.rb

```diff
@@ -41,19 +41,20 @@
         let(:format) { nil }
 
         context 'when the requested file exists in the blog/ directory' do
-          let(:show_content) { 'it is the best language ever' }
+          let(:show_content) { '</head>it is the best language ever' }
```

Comment:
Add `</head>` so that there is something for the `blog_html_with_additions` method to "hook into".